src/python.c

```diff
@@ -169,7 +169,7 @@ char *
 PyString_AsString(PyObject *unicode)
 {
 	char	   *rv;
-	PyObject   *o = PyUnicode_AsEncodedString(unicode, GetDatabaseEncodingName(), NULL);
+	PyObject   *o = PyUnicode_AsEncodedString(unicode, getPythonEncodingName(), NULL);
```

Comment:
I realise that  `getPythonEncodingName()` predates this commit, and my C is VERY rusty, but technically, is that function not buggy in returning the address of a local (i.e. temporary)? IMHO, since this commit will greatly increase the use of that function, it would be nice to move the string "ascii" to file scope, and return it's address to prevent an over-eager compiler messing things up. WDYT?

Reply:
I agree with your assessment.  I've moved the constant to a file scoped static and retested it.